src/main/resources/assets/computercraft/lua/rom/programs/advanced/multishell.lua

```diff
@@ -262,15 +282,26 @@ while #tProcesses > 0 do
         local button, x, y = tEventData[2], tEventData[3], tEventData[4]
         if bShowMenu and y == 1 then
             -- Switch process
-            local tabStart = 1
-            for n=1,#tProcesses do
-                local tabEnd = tabStart + string.len( tProcesses[n].sTitle ) + 1
-                if x >= tabStart and x <= tabEnd then
-                    selectProcess( n )
-                    redrawMenu()
-                    break
+            if x == 1 and nScrollpos ~= 1 then
+                nScrollpos = nScrollpos - 1
+                redrawMenu()
+            elseif x == term.getSize() and bScrollRight == true then
```

Comment:
remove "== true"

Reply:
Thinking optimisation, you may want to change the line to `elseif bScrollRight and x == term.getSize() then`. Doing this will mean that if bScrollRight then Lua will not bother calling the function to check if x is equal to the size of the terminal.